sdk/requirements.txt

```diff
@@ -4,9 +4,9 @@ fire
 google-auth>=1.30.0,<2.0dev
 google-cloud-storage>=1.37.0
 mlflow==1.14.1
```

Comment:
Can we relax the requirement of mlflow?

Reply:
Yep I've just relaxed the version of mlflow like in Merlin, as well as other dependencies like `google-auth` and `google-cloud-storage`.